docs/samples/explanation/alibi/alibiexplainer/README.md

```diff
@@ -17,9 +17,9 @@ After poetry has installed dependencies you should see:
 You can check for successful installation by running the following command
 
 ```
-usage: __main__.py [-h] [--http_port HTTP_PORT] [--grpc_port GRPC_PORT]
-                   [--predict_url PREDICT_URL] [--method {ExplainerMethod.anchor_tabular}]
-__main__.py: error: the following arguments are required: --predict_url
```

Comment:
do we need to delete options too?

Reply:
These options are not listed anymore if you run the main python file, so I've deleted it.

Reply:
e.g., this is printed instead, a lot more of options:
```bash
$ python alibiexplainer/__main__.py -h
usage: __main__.py [-h] [--http_port HTTP_PORT] [--grpc_port GRPC_PORT] [--workers WORKERS] [--max_threads MAX_THREADS] [--max_asyncio_workers MAX_ASYNCIO_WORKERS] [--enable_grpc ENABLE_GRPC] [--enable_docs_url ENABLE_DOCS_URL]   ─╯
                   [--enable_latency_logging ENABLE_LATENCY_LOGGING] [--configure_logging CONFIGURE_LOGGING] [--log_config_file LOG_CONFIG_FILE] [--access_log_format ACCESS_LOG_FORMAT] [--model_name MODEL_NAME]
                   [--predictor_host PREDICTOR_HOST] [--protocol {v1,v2,grpc-v2}] [--predictor_protocol {v1,v2,grpc-v2}] [--predictor_use_ssl PREDICTOR_USE_SSL]
                   [--predictor_request_timeout_seconds PREDICTOR_REQUEST_TIMEOUT_SECONDS] [--grpc_max_send_message_length GRPC_MAX_SEND_MESSAGE_LENGTH] [--grpc_max_receive_message_length GRPC_MAX_RECEIVE_MESSAGE_LENGTH]
```